include/palette.h

```diff
@@ -52,27 +69,37 @@ struct PaletteFadeControl
     u8 deltaY:4; // rate of change of blend coefficient
```

Comment:
I'm pretty sure the constant switching between `u16` u8` etc prevents these bits from actually being packed in a meaningful way, so reordering is probably in order

Reply:
Reordering done locally